aries-site/src/examples/components/avatar/AvatarDataTableExampleDo.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Avatar, Box, DataTable, ResponsiveContext, Text } from 'grommet';
+
+const data = [
```

Comment:
Can we bring this into our src/data/mockData folder? maybe have it called "users" or something

Reply:
done